docs/source/_installation/mac_silicon.rst

```diff
@@ -0,0 +1,104 @@
+Apple Silicon
+-------------
+Apple recently released it's new silicon for their latest lineup of products.
+While it is great to see more high performing quality chips in the market, it
+means there is work to be done ensuring all our favourite packages can
+utilize this new technology. At the time of writing this installation of
+most MDSuite dependencies is not too hard on a Mac M1, but we thought we would
+add some documentation just to be helpful.
+
+Getting the Source
+^^^^^^^^^^^^^^^^^^
+Unfortunately the Mac M1 install is currently only possible from source. To get started we need to get the source
+code from the repository. Open a terminal and run the following:
+
+.. code-block:: bash
+
+   git clone https://github.com/zincware/MDSuite.git
+   cd MDSuite
+
+This should download the MDSuite code and change you into the main directly where you will see the file structure.
+
+TensorFlow Install
+^^^^^^^^^^^^^^^^^^
+**The following is taken from issue #153 frpm the apple/tensorflow_macos repository found**
+`here <https://github.com/apple/tensorflow_macos/issues/153>`_
+
+In order to satisfy all of the dependencies we will need to use MiniForge to create a virtual environment.
```

Comment:
Could you give me a quick explanation on why to use MiniForge and not miniconda?

Reply:
No. They don't explain why they use this over conda explicitly. I suppose given that the installation is just as minimal that it could be possible to exchange the two.

Reply:
Could you also test conda? Because most people would prefer conda over miniforge I assume, because it is more common?

Reply:
```
miniforge started a few months ago because miniconda doens't support aarch64, very quickly the 'PyPy' people jumped on board, and in the mean time there are also miniforge versions for all Linux architectures, as well as MacOS.

Soon there will also be a windows variant (hopefully also for both CPython and PyPy)

I guess that an ARMv7 (32Bit ARM) variant is also on the horizon (Raspbian) 
```

Reply:
On the other hand, we might release it on conda eventually, so we can leave it as is for now

Reply:
Could also be the compilers

Reply:
I will give it a try though.